backend/daemon/api/entities/v1alpha/entities.go

```diff
@@ -355,3 +355,59 @@ func (api *Server) DiscoverEntity(ctx context.Context, in *entities.DiscoverEnti
 
 	return &entities.DiscoverEntityResponse{}, nil
 }
+
+// SearchEntities implements the Fuzzy search of entities.
+func (api *Server) SearchEntities(ctx context.Context, in *entities.SearchEntitiesRequest) (*entities.SearchEntitiesResponse, error) {
```

Comment:
@burdiyan Let's say I want to collect all the document's titles (the last version of each document). Should I call `LoadEntity` and then Hydrate the entity to get the doc's title? sounds like too much overhead. Should I create a custom function that traverses all document changes and gets the latest title?

Reply:
We should avoid reading blobs from disk entirely, and index the relevant values in the DB. Either as a column on `resources` table, or a separate table of some sort.

The trick is that we'd need to respect the CRDT rules to make sure we handle concurrent changes correctly. So if the author changes the title from two devices that aren't synced, we should handle the conflict and choose the last writer.

So maybe, instead of columns we could create a table like `resource_attrs` or something like that where we'd index relevant attributes, and store the blob ID that introduces the change.

We should be careful by only indexing changes from the owner (unless it's a group I guess, in which case I'm not sure what to do).